peer_mentoring/peer_mentoring/settings.py

```diff
@@ -13,8 +13,12 @@
 import os
 from pathlib import Path
 
+import environ
 from decouple import config
 
+env = environ.Env()
```

Comment:
is this another dependency? probably redundant, you can keep using decouple -> config